jwst/assign_wcs/util.py

```diff
@@ -111,7 +111,7 @@ def compute_scale(wcs: WCS, fiducial: Union[tuple, np.ndarray]) -> float:
 
     crpix = np.array(wcs.backward_transform(*fiducial))
     crpix_with_offsets = np.vstack((crpix, crpix + (1, 0), crpix + (0, 1))).T
-    crval_with_offsets = wcs(*crpix_with_offsets)
+    crval_with_offsets = wcs(*crpix_with_offsets, with_bounding_box=False)
```

Comment:
Perhaps instead of fixing this here, it would be better to fix it in whatever is calling this function in `outlier_detection` in the blotting code?  I suspect that it is using the wrong fiducial.  Instead of using the fiducial RA/Dec at the reference point of the distorted frame, it is using the RA/Dec reference point of the much larger rectified frame?

Reply:
Good point!